### ISSUE TITLE
lando drupal site:mode dev is failing

### ISSUE THREAD

**Issue by parijke** [lando/lando#977]
Bug Report
==========

**Tell us about your setup**

What is your lando version and operating system? **(note that the older your version the less likely we are to reply)**

v3.0.0-beta.45 on Ubuntu Linux 17.10

**Tell us about your `.lando.yml`**

Paste in the contents of your `.lando.yml` this is **SUPER HELPFUL** for us.
```
name: drupal-with-twig
recipe: drupal8
config:
  webroot: web
  drush: composer
  xdebug: true
```
**Tell us about the command you were running**

Paste in a series of commands that caused the bug.

lando drupal site:mode dev

**Tell us about the error you got**

[ERROR] Error copying file: /app/web/sites/default/services.yml   

I think it has something to do with the /app/ in the path? This is added by lando I think

**Comment by dustinleblanc** [comment]
@parijke `/app` is the root of your project within the appserver container.

This is what I get:

```
lando drupal site:mode dev


 [ERROR] Command "site:mode", is not a valid command name.
```

**Comment by parijke** [comment]
I think you must be using an older version then? It is a long time feature of drupal console. Here's the full output:
```
paul@paul-OptiPlex-790 ~/www/drupal_with_twig $ lando drupal site:mode dev
 Configuration name: system.performance
 ------------------------- ---------------- ---------------- 
  Configuration key         Original Value   Override Value  
 ------------------------- ---------------- ---------------- 
  cache.page.use_internal   false            false           
  css.preprocess            false            false           
  css.gzip                  false            false           
  js.preprocess             false            false           
  js.gzip                   false            false           
  response.gzip             false            false           
 ------------------------- ---------------- ---------------- 

 Configuration name: views.settings
 -------------------------------- ---------------- ---------------- 
  Configuration key                Original Value   Override Value  
 -------------------------------- ---------------- ---------------- 
  ui.show.sql_query.enabled        true             true            
  ui.show.performance_statistics   true             true            
 -------------------------------- ---------------- ---------------- 

 Configuration name: system.logging
 ------------------- ---------------- ---------------- 
  Configuration key   Original Value   Override Value  
 ------------------- ---------------- ---------------- 
  error_level         all              all             
 ------------------- ---------------- ---------------- 

                                                                                                                        
 [ERROR] Error copying file: /app/web/sites/default/services.yml                                                        
                                                                                                                        


 // cache:rebuild

 Rebuilding cache(s), wait a moment please.

                                                                                                                        
 [OK] Done clearing cache(s).                                                                                           
                                                                                                                        
```

**Comment by dustinleblanc** [comment]
On a different site I am able to reproduce.

Reading https://www.drupaleasy.com/quicktips/enabling-development-mode-local-drupal-8-site I can see an answer to both issues, it looks like they removed the command at some point, and it also looks like this is an issue with Drupal locking down sites/default. I was able to get around this by pulling a `chmod g+w` on the services.yml file.

Closing as this is a quirk with Drupal's permissions and Drupal Console, not Lando. Try the above to get around it 😄
